samples/SmokeTest/src/main/java/com/azure/CosmosDB.java

```diff
@@ -0,0 +1,127 @@
+package com.azure;
+
+import com.azure.data.cosmos.CosmosClient;
+import com.azure.data.cosmos.CosmosContainer;
+import com.azure.data.cosmos.CosmosItemProperties;
+import com.azure.data.cosmos.FeedResponse;
+import com.azure.data.cosmos.FeedOptions;
+import reactor.core.publisher.Flux;
+import reactor.core.scheduler.Schedulers;
+
+public class CosmosDB {
+    private static CosmosClient client;
+    private static CosmosContainer container;
+    private static final String dbName = "JavaSolarSystem";
+    private static final String collectionName = "Planets";
+
+    private static void createDatabase() {
+        System.out.print("Creating database... ");
```

Comment:
These should use our ClientLogger rather than System.out.println.

Reply:
Tried in pair programming and we weren't able to get it with azure level set to 0.

Reply:
Yeah, I wouldn't use ClientLogger here - the intent of this application (I believe) is to act as an end-user, who should never use the ClientLogger. This code should be replaced with direct use of the slf4j APIs. What we should do in this smoke test code however is include a slf4j logger dependency so that we can see all the logging output from the client libraries *and* the smoke test, and ensure it all appears as expected.